src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -111,7 +111,9 @@ if (__DEV__) {
     stopCommitLifeCyclesTimer,
   } = require('ReactDebugFiberPerf');
 
-  var warnAboutUpdateOnUnmounted = function(instance: ReactClass<any>) {
+  var warnAboutUpdateOnUnmounted = function(
+    instance: React$ComponentType<any>,
```

Comment:
Doesn't look right to me. `React$ComponentType` represents a React *type*, not instance. E.g. `MyClass` or `MyFunctionalComponent`.

The original typing was wrong too though.

Reply:
Thanks for catching this Dan. 👍